edunext_openedx_extensions/manage_api/urls.py

```diff
@@ -3,33 +3,20 @@
 
 """
 from django.conf.urls import url
-from .views import (
+
+from edunext_openedx_extensions.manage_api.views.enrollment import (
     UserManagement,
     OrgManagement,
     SubdomainManagement,
     OrganizationView,
 )
 
+from edunext_openedx_extensions.manage_api.views.users import PasswordManagement
 
 urlpatterns = [
-    url(
-        r'^v1/users/$',
-        UserManagement.as_view(),
-        name="manage_users_api"
-    ),
-    url(
-        r'^v1/organizations/$',
-        OrgManagement.as_view(),
-        name="manage_orgs_api"
-    ),
-    url(
-        r'^v1/subdomains/$',
-        SubdomainManagement.as_view(),
-        name="manage_subs_api"
-    ),
-    url(
-        r'^v1/edx-organizations/$',
-        OrganizationView.as_view(),
-        name="edx_orgs_api"
-    ),
+    url(r'^v1/users/$', UserManagement.as_view(), name="manage_users_api"),
+    url(r'^v1/organizations/$', OrgManagement.as_view(), name="manage_orgs_api"),
+    url(r'^v1/subdomains/$', SubdomainManagement.as_view(), name="manage_subs_api"),
+    url(r'^v1/edx-organizations/$', OrganizationView.as_view(), name="edx_orgs_api"),
+    url(r'^v1/change-password/$', PasswordManagement.as_view()),
```

Comment:
If we are going to keep this into its own class (see some comments below) it makes sense, that we put this into a namespace. We already made a mistake of leaving enrollment without the namespace.

By namespace I mean, we should group things into:
`v1/user/change-password/`. (it might be `user` or `user-administration` or something similar)

also at some point we could fix this for V2 and the enrollment endpoints:

```
url(r'^v2/enrollment/users/$', UserManagement.as_view(), name="manage_users_api"),
url(r'^v2/enrollment/organizations/$', OrgManagement.as_view(), name="manage_orgs_api"),
url(r'^v2/enrollment/subdomains/$', SubdomainManagement.as_view(), name="manage_subs_api"),
url(r'^v2/enrollment/edx-organizations/$', OrganizationView.as_view(), name="edx_orgs_api"),
url(r'^v2/user/change-password/$', PasswordManagement.as_view())
```